MUXControls.sln

```diff
@@ -850,20 +864,25 @@ Global
 		dev\SwipeControl\SwipeControl.vcxitems*{fd63f767-bd98-45d9-8a63-3cd4a148d527}*SharedItemsImports = 9
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug_test|Any CPU = Debug_test|Any CPU
```

Comment:
We should not have an "Any CPU" solution configuration. Please undo these changes.